### PR TITLE
Updated the test case

### PR DESCRIPTION
**Pull request by dmalarev**
The new test case checks if the server.broadcast() method is called
exactly once.

### CODE REVIEW

File: src/main/java/nl/tudelft/pixelperfect/event/EventLog.java [hunk 1]
```diff
@@ -72,7 +72,7 @@ public synchronized void notify(Event event) {
     String type = event.getClass().getSimpleName();
     EventsMessage eve =
```
Comment (jessetilro) [inline src/main/java/nl/tudelft/pixelperfect/event/EventLog.java]: Can you please just rename this variable to event?

**Comment by jessetilro** [comment]
FindBugs, PMD and CheckStyle all report a static error, can you fix this?

**Comment by dmalarev** [comment]
i'll check it out right away

**Comment by jessetilro** [comment]
Great. Could you test the EventLog's complete method while you're at it? You forgot this in PR #115.

**Comment by dmalarev** [comment]
Everything should now be added and tested.

**Comment by dmalarev** [comment]
working on the build failure

**Comment by dmalarev** [comment]
Right now, everything except Game and Scene has 100% line coverage

**Comment by jessetilro** [comment]
Nice work on the tests. I still get a FindBugs warning: IS2_INCONSISTENT_SYNC in EventLog on line 41.

**Comment by dmalarev** [comment]
Issue should be fixed now

**Comment by jessetilro** [comment]
Well done, everything is restored. Approved.

**Comment by dalderliesten** [comment]
Tests are clear, looks good to me.

**Pull request merged**